build/module-builder/package.json

```diff
@@ -57,6 +57,7 @@
   },
   "resolutions": {
     "fstream": ">=1.0.12",
-    "lodash": ">=4.17.19"
+    "lodash": ">=4.17.19",
+    "**/ssri": "^8.0.1"
```

Comment:
Are double asterisks (`**`) absolutely necessary to refer to any dependencies (e.g. a dependency's dependency)?

Reply:
> I think it would be better to use something like `"**/ssri": ">=8.0.1"`. This way, once ssri v9+ is released, it can be installed by any dependencies.

Idea her was to introduce the less amount of breaking changes by using the oldest major possible and then enabling upgrades on minors and patches. But yes this can be discussed I will investigate.


> Are double asterisks (`**`) absolutely necessary to refer to any dependencies (e.g. a dependency's dependency)?

From this [RFC](https://github.com/yarnpkg/rfcs/blob/master/implemented/0000-selective-versions-resolutions.md) **/ means "yarn will use <THE_PACKAGE> for every nested dependency to <THE_PACKAGE> and will behave as expected with respect to the node_modules folder by not duplicating the <THE_PACKAGE> installation."